docs/release_notes/v-1.5.md

```diff
@@ -0,0 +1,67 @@
+# Airbyte 1.5.0
+
+Valentine's Day let you down? That's OK. You'll love this. Airbyte 1.5.0 was released on February 19, 2025. We’re excited to share new improvements and changes to the Airbyte platform.
+
+## 🚀 Self-Managed Enterprise Changes
+
+These changes bring new capabilities to Airbyte's Self-Managed Enterprise customers.
+
+### Mappings in the UI
+
+We introduced mappings in the API in version 1.3, and it's now available in Airbyte's user interface, too. Use mappings to hash, encrypt, and rename fields, and filter rows. You set up mappings on each stream, ensuring your source data arrives in your destination in a more meaningful way. [**Learn more >**](../using-airbyte/mappings)
+
+![](../using-airbyte/images/mappings.png)
+
+### OpenTelemetry (OTel) metrics monitoring
+
+Airbyte now instruments a number of crucial metrics about syncs and data movement. If you use OTel as your observability framework, you can configure Airbyte to send this data to an OTel collector endpoint so you can consume these metrics in your monitoring tool of choice. The following metrics are available:
+
+- Sync details
+- Sync duration
+- Volume of data moved
+- API requests
+
+To start sending metrics, update your `values.yaml` file with the following configurations. [**Learn more >**](../operator-guides/collecting-metrics)
+
+```yaml title="values.yaml"
+global:
+    edition: enterprise # This is an enterprise-only feature
+    metrics:
+        enabled: true
+        otlp:
+            enabled: true
+            collectorEndpoint: "YOUR_ENDPOINT" # The OTel collector endpoint Airbyte sends metrics to. You configure this endpoint outside of Airbyte as part of your OTel deployment.
+```
+
+## 🚀 Platform Changes
+
+These changes improve Airbyte for all Self-Managed users.
+
+### Declarative OAuth 2.0 in the connector builder
```

Comment:
Can we include a Builder screenshot showing how the interface looks in the product?